ext/crates/sseq/src/coordinates/bidegree.rs

```diff
@@ -0,0 +1,103 @@
+use std::fmt::{self, Display, Formatter};
+use std::ops::{Add, AddAssign};
+
+use super::iter::{ClassicalIterator, StemIterator};
+
+/// Type synonym for (s, t) bidegrees.
+#[derive(Default, Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub struct Bidegree {
+    /// Homological degree
+    s: u32,
```

Comment:
I think we ought to work towards representing homological degree as a i32 instead of a u32. Then we can handle things like duals and homological shifts.

Reply:
I agree. That seems like a pretty intrusive change, so should I bundle it with this PR or open a new one?

Reply:
> open a new one?

Yeah in a separate PR would be better.